action/library/README.md

```diff
@@ -1,12 +1,15 @@
-# Fragments Actions Core
+# Action Library
+It is a core [actions](https://github.com/Knotx/knotx-fragments/tree/master/action#action) and
+[behaviours](https://github.com/Knotx/knotx-fragments/tree/master/action#behaviour) library.
 
-## HTTP Action
+## Action
+The core actions section.
```

Comment:
Maybe it would be worth to put some list of core actions/behaviours to have overview in one place (in form of a ToC).